src/files.js

```diff
@@ -980,5 +984,394 @@ module.exports = (common) => {
         )
       })
     })
+
```

Comment:
Can we break the MFS tests into another file? This test file is huge :)

Reply:
Yup!

Reply:
Do you think it should be added to index.js as a separate tests suit or should we just import them in files.js and there we would export them both?